packages/stream_video_flutter/android/src/main/kotlin/io/getstream/video/flutter/stream_video_flutter/service/PictureInPictureHelper.kt

```diff
@@ -41,11 +42,8 @@ class PictureInPictureHelper {
 
                     val params = PictureInPictureParams.Builder()
                     params.setAspectRatio(aspect).apply {
-                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
-                            setAutoEnterEnabled(true)
-                        }
                         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-                            setTitle("Video Player")
+                            setTitle("Video Call")
```

Comment:
Should we localize this? Where is the title shown?

Reply:
From the docs:
`Sets a title for the picture-in-picture window, which may be displayed by the system to give the user information about what this PIP is generally being used for.`

But I didn't see it anywhere while testing. It can be null so maybe I'll just remove it as it's not essential